src/Microsoft.IdentityModel.Tokens/SymmetricSignatureProvider.cs

```diff
@@ -262,7 +325,7 @@ public bool Verify(byte[] input, byte[] signature, int length)
             KeyedHashAlgorithm keyedHashAlgorithm = GetKeyedHashAlgorithm(GetKeyBytes(Key), Algorithm);
             try
```

Comment:
The Alloc() call is inside the try block and will cause the provider to be disposed if an exception occurs. On the other hand, a similar call in the SymmetricSignatureProvider is outside of a try block. Is this the intended behavior?